test/integration/selenium-helpers.js

```diff
@@ -68,7 +68,7 @@ class SeleniumHelper {
         let driverConfig = {
             browserName: 'chrome',
             platform: 'macOS 10.14',
-            version: '76.0'
```

Comment:
Maybe we should pull this from package.json?

Reply:
Oh, that's a good idea.